docs/plugins/security.asciidoc

```diff
@@ -0,0 +1,29 @@
+[[security]]
+== Security Plugins
+
+Security plugins add a security layer to  Elasticsearch.
+
+[float]
+=== Core security plugins
+
+The core security plugins are:
+
+link:/products/shield[Shield]::
+
+Shield is the Elastic product that makes it easy for anyone to add
```

Comment:
Same as for watcher. I think we should say here that it's a commercial product. As you did for Marvel BTW.